api/src/kots_app/kots_app_store.ts

```diff
@@ -67,6 +67,16 @@ export class KotsAppStore {
     await this.pool.query(q, v);
   }
 
+  async updateNamespace(id: string, namespace: string): Promise<void> {
+    const q = `update app set namespace = $1 where id = $2`;
```

Comment:
This column is used for registry namespace, not app namespace

Reply:
@divolgin do we currently have an app_downstream namespace column?

Reply:
Ah, do we need to add a new column for app namespace to this table or is that managed somewhere else?